robber/matchers/__init__.py

```diff
@@ -5,12 +5,12 @@
     'called_once',
     'called_once_with',
     'called_with',
-    'contain'
+    'contain',
     'equal',
-    'ever_called'
+    'ever_called',
```

Comment:
I know it's quite simple, but I prefer to follow the Python zen in this case. What I'm thinking that we should do the same thing to the other apis, to keep them pythonic, 1 obvious way for every case.